torch_xla/launch.py

```diff
@@ -0,0 +1,49 @@
+import warnings
+
+import torch.distributed as dist
+import torch_xla
+import torch_xla.core.xla_env_vars as xenv
+import torch_xla.distributed.xla_multiprocessing as xmp
+import torch_xla.utils.utils as xu
+import torch_xla.runtime as xr
+
+
+def spawn_wrapper(index, fn, *args, **kwargs):
+  dist.init_process_group('xla', init_method='xla://')
+  fn(index, *args, **kwargs)
+
+
+def launch(
+    fn,
+    *args,
+    **kwargs,
```

Comment:
`spawn` does not actually take `kwargs`, so for consistency, you should not take `kwargs` for `fn` in this wrapper. https://pytorch.org/docs/stable/multiprocessing.html#spawning-subprocesses

Also, I don't recommend mixing variable-length arguments for both `launch` and `fn`. You can get around that by making `args` a singular tuple, similar to `spawn`.

Reply:
Cleaned up, it's now much simpler.